src/Tickets/Flexible_Tickets/Series_Passes/Series_Passes.php

```diff
@@ -1158,19 +1158,21 @@ public function allow_tickets_on_recurring_events( bool $allow ): bool {
 	 *
 	 * @since 5.8.0
 	 *
-	 * @param WP_Rewrite $wp_rewrite Current WP_Rewrite instance (passed by reference).
+	 * @since TBD Filter my tickets rewrite rules to include the series page.
 	 *
-	 * @return void
+	 * @param array $rules Rewrite rules array.
+	 *
+	 * @return array The updated rewrite rules array.
```

Comment:
Add the `array<T>` type, it _should_ be `array<string>` or `array<string,string>`